### ISSUE TITLE
Specify Order By 

### ISSUE THREAD

**Issue by zacksayenko** [neo4j-graphql/neo4j-graphql-js#47]
Is there a way to specify Order By for arrays of nodes?
I see the current generated cypher queries do not have Order By specified and it just returns the list of nodes in a default order, is there a way to change it?

**Comment by appinteractive** [comment]
Any progress on this?

**Comment by appinteractive** [comment]
@johnymontana is any help needed or @zacksayenko is there any workaround we can use for now? Its really important for me as I'm in the middle of a product launch :)

**Comment by appinteractive** [comment]
Yeah @johnymontana but I always get the default order. Im ordering the results by creation date which I saved as string. Could this be the problem? Do I have to do any atitional stuff on the database level?

<img width="1606" alt="grafik" src="https://user-images.githubusercontent.com/432828/47718943-01c24680-dc4a-11e8-8986-51cd6a27b7f4.png">

Here the query that was run:
```
MATCH (user:User {slug:$slug,deleted:$deleted,disabled:$disabled}) 
RETURN user { .name ,contributions: [(user)-[:WROTE]->(user_contributions:Post{}) | user_contributions { .createdAt }][0..10] } AS user SKIP $offset
{ offset: 0,
  first: -1,
  slug: 'sabine-klee',
  deleted: false,
  disabled: false,
  '1_first': 10,
  '1_offset': 0,
  '1_orderBy': 'createdAt_desc' }
```

**Comment by johnymontana** [comment]
@appinteractive after looking into this I realized we've only implemented ordering at the top level operation and not nested selections. This is a bug as the `orderBy` argument is added, but ordering is not actually implemented in the generated query. We'll fix this soon!

**Comment by appinteractive** [comment]
@johnymontana thanks! Can you specifiy "soon" a bit more? Just want to know as we depend on that feature in our project.

**Comment by johnymontana** [comment]
@appinteractive we've prioritized it for the current sprint. I'll work on moving all issues into a Github project board so we have better visibility of the roadmap and current progress

**Comment by smkhalsa** [comment]
@appinteractive any progress on this?

**Comment by GeoffThorpeFT** [comment]
@johnymontana Can you give us any progress update on the nested orderBy?

**Comment by appinteractive** [comment]
@johnymontana yeah any progress? Happy New Year by the way :) 🎉🥳

**Comment by GeoffThorpeFT** [comment]
@johnymontana. Its been about a month since the last chase up .... any progress re a fix to this issue?

**Comment by johnymontana** [comment]
The fix for nested ordering should be dropping this week - sorry for the delay on this.

**Comment by johnymontana** [comment]
The fix for this is now available in v2.3.0 - please test and let us know how it works.

**Comment by jonasdumas** [comment]
The problem is still there. We've nested items :
```
Post(ttuid: $postTtuid){
      comments(orderBy: created_at_desc){
          ...Comment
      }
  }
```

Whetever field we define for orderBy to order our comments, it doesn't change the order.

We've version neo4j-graphql-js 2.11.0

Any idea why?
